test/rulesets.jl

```diff
@@ -18,6 +18,9 @@ end
 
 @testset "Numeric" begin
     @syms a::Integer b c d x::Real y::Number
+    @eqtest simplify(conj(x)) == x
+    @eqtest simplify(real(x)) == x
+    @eqtest simplify(imag(x)) == 0
```

Comment:
You will need to update the tests because `conj(x)` may just return `x`. Use `Term{Real}(conj, [x])` instead.

Reply:
Ok, ok.